src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -12,37 +12,62 @@
                         :initial-branches="initialBranches"
                         @branchSelected="branchSelected"
                         @commitSelected="commitSelected"
-                        @reportsUpdate="updateReports"
+                        @reportsUpdate="updateAvailableReports"
                     ></git-update-reports>
                 </div>
+                <b-alert
+                    :show="!!workflowRemovals"
+                    dismissible
+                    variant="warning"
+                    @dismissed="workflowRemovals=null">
+                    The following items are not present in this git commit and have been removed from the workflow:
+                    <ul class="py-0 my-0 ml-2" :style="{listStyleType: 'disc'}">
+                        <li v-for="item in workflowRemovals">{{item}}</li>
+                    </ul>
+                </b-alert>
             </div>
-            <div class="pb-4">
+            <div class="pb-4" id="workflow-reports">
                 <h2 id="report-sub-header">Reports</h2>
                 <div>
-                    <div id="preprocess-div" class="form-group row">
-                        <label class="col-sm-2 col-form-label text-right">Preprocess</label>
-                        <div>
-                        <div class="form-group row">
-                            <label for="n-min" class="col-sm-4 col-form-label text-right">nmin:</label>
-                            <div class="col-sm-8 input-group">
-                                <input type="text" class="form-control mr-2" id="n-min" placeholder="7">
-                                <button id="workflow-remove-button" type="button" class="px-2 btn btn-primary">Remove report</button>
-                            </div>
-                        </div>
-                        <div class="form-group row">
-                            <label for="n-max" class="col-sm-4 col-form-label text-right">nmax:</label>
-                            <div class="col-sm-4">
-                                <input type="text" class="form-control" id="n-max" placeholder="16">
-                            </div>
+                    <div v-for="(report, index) in workflowMetadata.reports"
+                         :id="`workflow-report-${index}`"
+                         :key="index"
+                         class="form-group row">
+
+                        <label class="col-sm-2 col-form-label text-right">{{report.name}}</label>
+                        <parameter-list
+                            v-if="reportParameters[index].length > 0"
+                            :params="reportParameters[index]"
+                            @paramsChanged="(...args) => paramsChanged(index, ...args)"
```

Comment:
I think ...args should be replaced for a more descriptive parameter.

Reply:
It is just the event args, so I've changed it to `eventArgs` in hopes that's clearer. I'd rather not use `params` or `parameters` since this event is all about changing report parameters, so I think that would be more confusing. Let me know if you can think of anything that would be clearer though.